src/test/java/uk/gov/ch/controller/update/EntityDataControllerTest.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.ch.controller.update;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import uk.gov.ch.exception.EntityEmailNotFoundException;
+import uk.gov.ch.model.update.EntityData;
+import uk.gov.ch.update.impl.EntityDataServiceImpl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class EntityDataControllerTest {
+
+    @Mock
+    private EntityDataServiceImpl entityDataService;
+
+    @InjectMocks
+    private EntityDataController entityDataController;
+
+    private static final String INCORPORATION_NUMBER = "12345678";
```

Comment:
oe123456

Reply:
This is just test data, but if we want something which reflects Production I can change no problem.